setup.cfg

```diff
@@ -1,9 +1,124 @@
-[versioneer]
-VCS = git
-style = pep440-post
-versionfile_source = ophyd/_version.py
-versionfile_build = ophyd/_version.py
-tag_prefix = v
+[metadata]
+name = ophyd
+description = Bluesky hardware abstraction with an emphasis on EPICS
+url = https://github.com/bluesky/ophyd
+license = BSD
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+classifiers =
+    License :: OSI Approved :: BSD License
+    Development Status :: 5 - Production/Stable
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+
+[options]
+python_requires = >=3.8
+packages = find:  
+setup_requires =
+    setuptools_scm[toml]>=6.2
+
+# Specify any package dependencies below.
+install_requires =
+    networkx>=2.0
+    numpy
+    packaging
+    pint
+
+[options.extras_require]
+# For development tests/docs
+dev =
+    attrs>=19.3.0
+    black==22.3.0
+    bluesky>=1.7.0
+    caproto[standard] >=0.4.2rc1
+    codecov
+    databroker>=1.0.0b1
+    doctr
+    epics-pypdb
+    # Upper bound on flake8 while flake8-isort catches up
+    # https://github.com/gforcada/flake8-isort/issues/115
+    flake8<5.0.0
+    flake8-isort
+    h5py
+    inflection
+    ipython
+    ipywidgets
+    matplotlib
+    mypy
+    myst-parser
+    numpydoc
+    pre-commit
+    pyepics>=3.4.2
+    pytest
+    pytest-cov    
+    pytest-faulthandler
+    pytest-rerunfailures
+    pytest-timeout
+    pipdeptree
+    setuptools_scm[toml]>=6.2
+    sphinx>=1.8.1,!=2.0.0
+    sphinx_rtd_theme
+    tox
+
+# Specify any package data to be included in the wheel below.
+[options.package_data]
+ophyd =
+    # Include our documentation helpers:
+    "*.rst"
+
+[options.entry_points]
+# Include a command line script
+console_scripts =
+    python3-pip-skeleton = python3_pip_skeleton.__main__:main
```

Comment:
I'm not super familiar with these files, what exactly is this doing? Naively it looks to me like ophyd is providing a console script called `python3-pip-skeleton` defined in a different library?

Reply:
Oops, that's a copy-paste fail, there's not meant to be a console_script for ophyd. Deleted in 9f0847e